web/src/components/apps/ClusterNodes.jsx

```diff
@@ -39,12 +39,18 @@ export class ClusterNodes extends Component {
     }
   }
 
-  generateWorkerAddNodeCommand = () => {
+  generateWorkerAddNodeCommand = async () => {
     this.setState({ generating: true, command: "", expiry: null });
 
-    this.props.generateWorkerAddNodeCommand()
-      .then((resp) => {
-        const data = resp.data.generateWorkerAddNodeCommand;
+    fetch(`${window.env.API_ENDPOINT}/kurl/generate-node-join-command-worker`, {
```

Comment:
This might be the old way of locating the API. I think Jelena updated everything to use window.location.